ui/app/adt/views/wardList.html

```diff
@@ -1,38 +1,43 @@
 <div class="ward-details-container">
-    <input type="text" ng-model="searchText" placeholder="{{'IPD_WARD_LIST_SEARCH_KEY'| translate }}"/>
+    <input type="text" ng-model="searchText" placeholder="{{'IPD_WARD_LIST_SEARCH_KEY'| translate }}" />
     <span ng-if="tableDetails.length == 0" class="no-data-info">
         {{ 'EMPTY_WARD_MESSAGE'|translate }}
     </span>
     <div style="overflow-x: scroll">
         <table class="table ward-details-content" ng-if="tableDetails.length > 0">
             <thead>
-            <tr>
-                <th ng-repeat="heading in tableHeadings" ng-if="heading != 'hiddenAttributes'" ng-class="{'bed-detail-column' : $index == 0}">{{heading | titleTranslate}}</th>
-            </tr>
+                <tr>
+                    <th ng-repeat="heading in tableHeadings" ng-if="heading != 'hiddenAttributes'"
+                        ng-class="{'bed-detail-column' : $index == 0}">{{heading | titleTranslate}}</th>
+                </tr>
             </thead>
             <tbody>
-            <tr ng-repeat="row in tableDetails | filter:searchTextFilter">
-                <td ng-repeat="heading in tableHeadings" ng-if="heading != 'hiddenAttributes'">
-                    <div ng-if="heading != 'Diagnosis' && heading != 'Id'">
-                        {{row[heading]}}
-                        <span ng-if="::(row[iconAttributeConfig.attrName] === iconAttributeConfig.attrValue && heading === 'Name')" class="icons-container">
-                            <i class="ipd-indication" ng-class="iconAttributeConfig.icon" ng-style="{{iconAttributeConfig.iconStyle}}"></i>
-                        </span>
-                    </div>
-                    <a ng-if="heading =='Id'" ng-click="gotoPatientDashboard(row['hiddenAttributes'].patientUuid, row['hiddenAttributes'].visitUuid)">
-                        {{row[heading]}}
-                    </a>
-                    <div ng-if="heading == 'Diagnosis'">
-                        <div ng-repeat="diagnosis in row[heading]" ng-class="{'strike-text': diagnosis.ruledOut}">
-                            <span>{{diagnosis.Diagnosis}}</span>
-                            <span>{{diagnosis["Diagnosis Certainty"]}}</span>
-                            <span>{{diagnosis["Diagnosis Order"]}}</span>
-                            <span ng-if="diagnosis.diagnosisStatus">{{diagnosis.diagnosisStatus}}</span>
+                <tr ng-repeat="row in getSortedTableDetails() | filter:searchTextFilter">
+                    <td ng-repeat="heading in tableHeadings" ng-if="heading != 'hiddenAttributes'">
+                        <div ng-if="heading != 'Diagnosis' && heading != 'Id'">
+                            {{row[heading]}}
+                            <span
+                                ng-if="::(row[iconAttributeConfig.attrName] === iconAttributeConfig.attrValue && heading === 'Name')"
+                                class="icons-container">
```

Comment:
@vikramdasvaishnav-thoughtworks i don't see any change in this file except the space changes. can you remove this file from commit if there are no changes